src/routes/RequestsRoute.js

```diff
@@ -563,6 +568,7 @@ class RequestsRoute extends React.Component {
     const patronGroups = (resources.patronGroups || {}).records || [];
     const addressTypes = (resources.addressTypes || {}).records || [];
     const servicePoints = (resources.servicePoints || {}).records || [];
+    const cancellationReasons = get(resources, 'cancellationReasons.records', []);
```

Comment:
Can we keep the same structure for all resources here, i.e. use 
```
get(resources, 'resourceName.records', []);
```
or 
```
(resources.resourceName || {}).records || [];
```
across the board?

Reply:
Yes I was thinking the same but felt it's out of this PR. I think it would be good to do all sort of cleanup in ui-requests and ui-inventory. I will update these now.